### DISCUSSION TITLE
Limit number of names on model list?

### DISCUSSION THREAD

**Discussion by Shingotink** [UltimaHoarder/UltimaScraper#1168] [Q&A]
Is there a way to limit the number of names that are searched for?

The list seems to sort models by expiration time, and it the more subscriptions you have the longer it takes to create your "Names" list (for me it's currently taking ~30min each scrape).

Is there a way to setup to list say only the first 50 names for this list? That way I can focus on models who will be expiring first.

Having an option for a "short list" and a "full list" would be awesome too.

**Comment by LunarPenguin** [comment]
I don't think such functionality exists at the moment (you could create an issue requesting the functionality if you want). A workaround is to put the models you don't want to scrape at the moment in the blacklist. this prevents them from showing up in the names list.
